TestObjectiveDropbox/TestObjectiveDropbox_iOSTests/FileRoutesTests.m

```diff
@@ -65,7 +78,7 @@ - (void)tearDown {
 
 - (void)testFileRoutes {
     XCTestExpectation *flag = [[XCTestExpectation alloc] init];
-    [_tester testFilesEndpoints:^{
+    [_tester testAllUserAPIEndpoints:^{
```

Comment:
is this a typo? should this be `testFilesEndpoints`?

Reply:
no, this is corrrect. testAllUserAPIEndpoints runs testFilesEndpoints and sharing tests